algorithms/corpus_creators/basic.py

```diff
@@ -50,3 +65,35 @@ def _transform(self, document):
             self.nlp.lowercase_strip_punc(document[field])
             for field in self.document_schema_fields
         ])
+
+class GensimCorpusCreator(CorpusCreator):
+    """
+        An object that transforms job listing documents by picking
+        important schema fields and returns them as one large cleaned array of words
+    """
+    document_schema_fields = [
+        'description',
+        'experienceRequirements',
+        'qualifications',
+        'skills'
+    ]
+    join_spaces = ' '.join
+
+    def _transform(self, document):
+        return self.join_spaces([
+            self.nlp.clean_str(document[field])
+            for field in self.document_schema_fields
+        ])
+
+class JobCategoryCorpusCreator(CorpusCreator):
+    """
+    Fake job category labeler
```

Comment:
How is this one fake? The name of the class makes sense, but the docstring is confusing.